doc/whats_new.rst

```diff
@@ -14,7 +14,8 @@ Current
 
 Changelog
 ~~~~~~~~~
-
+- :func:`fit_trans_from_points` allows retrieving a :code:`trans` object from a BIDS dataset that contains MEG and T1 weighted MRI data, by `Stefan Appelhoff`_ (`#211 <https://github.com/mne-tools/mne-bids/pull/211>`_)
```

Comment:
do you expect users to call fit_trans_from_points directly? I would say not so I would not expose the function publicly

Reply:
What is your alternative proposal? Call `fit_trans_from_points` in `read_raw_bids` and add an argument and an optional return value to `read_raw_bids`?